.vscode/cspell.json

```diff
@@ -2,7 +2,11 @@
   "version": "0.1",
```

Comment:
Why did this formatting get triggered? Can you undo the formatting for this json or maybe move to another PR?